.github/workflows/deploy.yml

```diff
@@ -0,0 +1,106 @@
+name: Deploy
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Metabase version to deploy'
+  workflow_run:
+    workflows: ["Shoppy E2E tests"]
+    types:
+      - completed
+    branches:
+      - "release"
+
+env:
+  version: ${{ github.event.inputs.version || vars.MB_VERSION }}
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' }} || github.event_name == 'workflow_dispatch' }}
```

Comment:
The typo here.

Should be `${{ github.event.workflow_run.conclusion == 'success' || github.event_name == 'workflow_dispatch' }}`

Reply:
Fixed